templates/juju/charmhub-community/index.html

```diff
@@ -0,0 +1,367 @@
+                            attrs={
+                            "class": "p-juju-community-tabs__logo u-no-margin--bottom",
+                            },) | safe
+              }}
+            </a>
+            <h5 class="p-heading--5 u-no-margin--bottom">
+              <a href="https://charmhub.io/prometheus-k8s">Prometheus</a>
+            </h5>
+            <p>Open source monitoring system with a dimensional data model and a time series database</p>
+          </div>
+          <div class="grid-col-2">
+            <a href="https://charmhub.io/grafana-k8s">
+              {{ image(url="https://assets.ubuntu.com/v1/1c10fa01-grafana-logo-jujuis.png",
+                            alt="",
+                            height="288",
+                            width="231",
+                            hi_def=True,
+                            loading="auto",
+                            attrs={
+                            "class": "p-juju-community-tabs__logo u-no-margin--bottom",
+                            },) | safe
+              }}
+            </a>
+            <h5 class="p-heading--5 u-no-margin--bottom">
+              <a href="https://charmhub.io/grafana-k8s">Grafana</a>
+            </h5>
+            <p>Open source analytics and interactive visualization web application</p>
+          </div>
+          <div class="grid-col-2">
+            <a href="https://charmhub.io/loki-k8s">
+              {{ image(url="https://assets.ubuntu.com/v1/a647636f-loki-logo-jujuis.png",
+                            alt="",
+                            height="288",
+                            width="211",
+                            hi_def=True,
+                            loading="auto",
+                            attrs={
+                            "class": "p-juju-community-tabs__logo u-no-margin--bottom",
+                            },) | safe
+              }}
+            </a>
+            <h5 class="p-heading--5 u-no-margin--bottom">
+              <a href="https://charmhub.io/loki-k8s">Loki</a>
+            </h5>
+            <p>
+              Open source log aggregation system designed to store and query logs from all your applications and infrastructure
+            </p>
+          </div>
+        </div>
+        <div class="grid-row u-hide" id="infrastructure">
+          <div class="grid-col-2">
+            <a href="https://charmhub.io/vault">
+              {{ image(url="https://assets.ubuntu.com/v1/8f9d47a0-vault-logo-jujuis.png",
+                            alt="",
+                            height="288",
+                            width="262",
+                            hi_def=True,
+                            loading="auto",
+                            attrs={
+                            "class": "p-juju-community-tabs__logo u-no-margin--bottom",
+                            },) | safe
+              }}
+            </a>
+            <h5 class="p-heading--5 u-no-margin--bottom">
```

Comment:
@muhammad-ali-pk sorry one more, we're skipping heading hierarchy with these h5's, since it looks like the highest prev heading is h2 could you please update all to be h3s styled as h5s? cc: @immortalcodes